lib/driver.js

```diff
@@ -473,10 +473,7 @@ class AndroidDriver extends BaseDriver {
   }
 
   getPrefsBuilder () {
```

Comment:
ah ok, if it's to make it testable than i'm fine with it. maybe just add a comment to that effect?